### PR TITLE
Use Octokit cache feature

### PR DESCRIPTION
**Pull request by minuscorp**

### CODE REVIEW

File: app/features/build_runner/fastlane_build_runner.rb [hunk 1]
```diff
@@ -105,9 +105,10 @@ def run(completion_block: nil, &block)
         logger.info("fastlane run complete")
         logger.debug(build_output.join("\n").to_s)
 
-        # TODO: need real artifacts here, are they artifacts or artifact paths?
```
Comment (taquitos) [inline app/features/build_runner/fastlane_build_runner.rb]: looks like this is duplicate changes from your other PR around artifacts

File: app/shared/models/git_repo.rb [hunk 2]
```diff
@@ -60,6 +61,18 @@ class GitRepo
 
     class << self
       attr_accessor :git_action_queue
+
+      # Loads the octokit cache stack for speed-up calls to github service.
+      # As explained in: https://github.com/octokit/octokit.rb#caching
+      def load_octokit_cache_stack
```
Comment (taquitos) [inline app/shared/models/git_repo.rb]: love the comments, and I think this is a great addition.

File: app/features/build_runner/fastlane_build_runner.rb (hunk 1 above)
Comment (minuscorp) [reply app/features/build_runner/fastlane_build_runner.rb]: I don't know w** is happening with my git today

**Comment by minuscorp** [comment]
Let me split this in a moment D:

**Comment by taquitos** [comment]
Haha

**Pull request closed without merging**